discovery/azure/azure.go

```diff
@@ -21,8 +21,9 @@ import (
 	"strings"
 	"time"
 
-	"github.com/Azure/azure-sdk-for-go/arm/compute"
-	"github.com/Azure/azure-sdk-for-go/arm/network"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
```

Comment:
The latest version is `2018-10-01` could we use that ?

Reply:
Sure. Thanks @sylr